### ISSUE TITLE
Not recognising one setup

### ISSUE THREAD

**Issue by Brandieee** [LlemonDuck/one-click-summer-garden#1]
![image](https://user-images.githubusercontent.com/10778583/105983144-91a90200-6098-11eb-8e27-96023fa743d9.png)

This placement of the elementals works to run through in one click but seemingly doesn't get recognised by the plugin

Worth mentioning I switched around the colours
![image](https://user-images.githubusercontent.com/10778583/105983465-1562ee80-6099-11eb-9d88-4b4ad8025c43.png)

**Comment by LlemonDuck** [comment]
There are definitely other configurations that work, but I found them to be
less consistent.

Particularly for E1 and E3 in that screenshot, they're just off-tick
relative to the global tick counter, and the rest of the elementals are
based off them. I found that aligning them to the global tick counter
helped with reducing input delay when clicking the tree (something to do
with models not lining up to their true positions maybe?).

If you were to reset all the elementals in the entire garden by just 1
tick, that would solve the above, and maybe also be recognized by the
plugin.
